storage/store.go

```diff
@@ -1491,6 +1491,12 @@ func (s *Store) ProposeRaftCommand(idKey cmdIDKey, cmd proto.RaftCommand) <-chan
 	return s.multiraft.SubmitCommand(cmd.RangeID, string(idKey), data)
 }
 
+// coincideRaftLeader try to make raft leader located in the node which
+// range leader located.
+func (s *Store) coincideRaftLeader(rangeID proto.RangeID) error {
```

Comment:
Related to early comments: `coincideRaftLeader` is the correct name for the store package, so this is fine.